deps/rabbitmq_auth_backend_http/examples/rabbitmq_auth_backend_spring_boot_kotlin/src/main/resources/rabbitmq.conf

```diff
@@ -4,6 +4,8 @@
 
 # http backend
 auth_backends.1 = rabbit_auth_backend_http
+auth_http.timeout     = 10000
+auth_http.connect_timeout     = 5000
```

Comment:
Let's rename this to `auth_http.connection_timeout`. In fact, that's what [`httpc`'s own doc reference](http://erlang.org/doc/man/httpc.html#set_options-1) calls this timeout :/